### PR TITLE
fix: polyfill intl for android

### LINKED ISSUE #2037: Cannot find variable Intl
### Bug description

Completing onboarding leads to a crash because of a missing variable

### Environment Specs

- Manufacturer: Android Studio
- Model: Emulator

- Trinity version: 924f9b6b69380c2351efdc3f3e8a63d0566532cc

### Steps To Reproduce
1. Complete onboarding
2. Observe loading screen
3. Once loading finished, the application will crash

### Expected behaviour
The application should not crash after loading

### Actual behaviour
The application crashed after loading

### Errors

Screenshot 2019-07-16 at 2 13 48 am

### PR DESCRIPTION
**Pull request by laumair**
# Description

Fixes https://github.com/iotaledger/trinity-wallet/issues/2037

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

# How Has This Been Tested?

- Manually tested Android emulator with english (default) selected language

# Checklist:

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] For changes to `mobile` that include native code (including React Native modules): I have verified that both iOS and Android successfully build in both `Debug` and `Release` modes
- [ ] For changes to `shared`: If applicable, I have verified that my changes are implemented correctly in `desktop` and `mobile`

### CODE REVIEW

File: src/mobile/index.js [hunk 1]
```diff
@@ -1,4 +1,13 @@
 import { AppRegistry } from 'react-native';
 import Wallet from './src/ui/routes/entry';
+import { isAndroid } from './src/libs/device';
+
+if (isAndroid) {
+    // This Intl API is not available in android 
+    // It needs to be polyfilled
+    require('intl');
+
+    require('intl/locale-data/jsonp/en-US');
```
Comment (laumair) [inline src/mobile/index.js:10]: @rajivshah3 I guess we might need to require all supported locales for Trinity here?

Comment (rajivshah3) [reply src/mobile/index.js:10]: Is the `require('intl')` sufficient? The readme says this:

> In Node.js, or when using require('intl'), the data is automatically added to the runtime and does not need to be provided.

https://www.npmjs.com/package/intl#locale-data

**Review by rajivshah3 — APPROVED** [review]

File: src/mobile/index.js (hunk 1 above)
Comment (laumair) [reply src/mobile/index.js:10]: No it is not. Doesn't work if I remove this import.

Comment (rajivshah3) [reply src/mobile/index.js:10]: Oh ok 🤔 yeah I guess we'll need to add all supported locales. Are we planning to release an update before #2012 is merged?

Comment (laumair) [reply src/mobile/index.js:10]: I don't think so. Perhaps we should merge this then to fix the crash?

**Pull request merged**